src/main/java/com/ifmo/epampractice/entity/UsersEntity.java

```diff
@@ -0,0 +1,193 @@
+    }
+
+    public UsersEntity(int id, String email, String passwordHash, UserRole role, String name,
+                       LocalDate birthDate, LocalDateTime signUpDate, boolean isBlocked,
+                       BigDecimal reputation) {
+        this.id = id;
+        this.email = email;
+        this.passwordHash = passwordHash;
+        this.role = role;
+        this.name = name;
+        this.birthDate = birthDate;
+        this.signUpDate = signUpDate;
+        this.isBlocked = isBlocked;
+        this.reputation = reputation;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public String getPasswordHash() {
+        return passwordHash;
+    }
+
+    public void setPasswordHash(String passwordHash) {
+        this.passwordHash = passwordHash;
+    }
+
+    public UserRole getRole() {
+        return role;
+    }
+
+    public void setRole(UserRole role) {
+        this.role = role;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public LocalDate getBirthDate() {
+        return birthDate;
+    }
+
+    public void setBirthDate(LocalDate birthDate) {
+        this.birthDate = birthDate;
+    }
+
+    public LocalDateTime getSignUpDate() {
+        return signUpDate;
+    }
+
+    public void setSignUpDate(LocalDateTime signUpDate) {
+        this.signUpDate = signUpDate;
+    }
+
+    public int getPassId() {
+        return passId;
+    }
+
+    public void setPassId(int passId) {
+        this.passId = passId;
+    }
+
+    public int getDrivingLicenseId() {
+        return drivingLicenseId;
+    }
+
+    public void setDrivingLicenseId(int drivingLicenseId) {
+        this.drivingLicenseId = drivingLicenseId;
+    }
+
+    public String getContactPhone() {
+        return contactPhone;
+    }
+
+    public void setContactPhone(String contactPhone) {
+        this.contactPhone = contactPhone;
+    }
+
+    public String getAddress() {
+        return address;
+    }
+
+    public void setAddress(String address) {
+        this.address = address;
+    }
+
+    public boolean isBlocked() {
+        return isBlocked;
+    }
+
+    public void setBlocked(boolean blocked) {
+        isBlocked = blocked;
+    }
+
+    public BigDecimal getReputation() {
+        return reputation;
+    }
+
+    public void setReputation(BigDecimal reputation) {
+        this.reputation = reputation;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        UsersEntity that = (UsersEntity) o;
+
+        return Objects.equals(id, that.id) &&
```

Comment:
Is int object in java? Use == for basic types and Objects.equals() for Objects

Reply:
fixed